dash/backend/src/migrations/2023062816298482489_falco_rule_join_tables.ts

```diff
@@ -0,0 +1,25 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<any> {
+    await knex.schema.alterTable('falco_rules', (table) => {
```

Comment:
We need to move existing data into the new tables, so maybe it needs to be: 
 - create new tables
 - copy old data into new tables (insert into ... select *)
 - then drop the old columns

Reply:
The other alternative would just be to use an array[] data type or the cluster_ids fields

Reply:
I didn't think it was worth doing this.

Reply:
(As the only DB with data in this table is my local copy)